src/Collection.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace OpenFoodFacts;
+
+use OpenFoodFacts\Document;
```

Comment:
this import is unused, please remove it

Reply:
This import is used at line 32.

Reply:
My bad 😛

Reply:
Wait a minute, it's the same namespace, so no import is needed